### PR TITLE
StatObejct  Integration Tests[Read only file system]

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
StatObject Tests
-  statExistingObject - File, Dir , FileFromBucketDirectory, SubDirectory
-  statNonExistingObject - File, Dir , FileFromBucketDirectory, SubDirectory

### Testing details
1. Manual - Tested all the tests are passing
2. Unit tests - N/A
3. Integration tests - All the inetgration tests are passing from integration tests directory

### CODE REVIEW

File: tools/integration_tests/readonly/stat_object_test.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+	file := statObject(filePath, t)
+
+	// Name - testBucket/Test1.txt, Type - File
+	if file.Name() != FileNameInTestBucket || file.IsDir() != false {
+		t.Errorf("Object stated for file in bucket is incorrect.")
+	}
+}
+
+func TestStatFileFromBucketDirectory(t *testing.T) {
```
Comment (sethiay) [inline tools/integration_tests/readonly/stat_object_test.go:45]: does TestStatFileInBucketDirectory sounds better ?

File: tools/integration_tests/readonly/stat_object_test.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+	dir := statObject(dirPath, t)
+
+	// Name - testBucket/Test/, Type - Dir
+	if dir.Name() != DirectoryNameInTestBucket || dir.IsDir() != true {
+		t.Errorf("Object stated for bucket directory is incorrect.")
+	}
+}
+
+func TestStatSubDirectory(t *testing.T) {
```
Comment (sethiay) [inline tools/integration_tests/readonly/stat_object_test.go:67]: Shall we also stat File in SubDirectory ?

**Review by sethiay — COMMENTED** [review]
Left Minor comments

File: tools/integration_tests/readonly/stat_object_test.go (hunk 2 above)
Comment (Tulsishah) [reply tools/integration_tests/readonly/stat_object_test.go:67]: We can but I thought stating file from directory will cover all cases. 
although I will include

**Comment by Tulsishah** [comment]
> Left Minor comments

Fixed

**Review by sethiay — APPROVED** [review]

**Pull request merged**